src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -329,11 +331,13 @@ class CommandRequestImpl : public CommandImpl,
       const mobile_apis::AppHMIType::eType app_hmi_type) const;
 
   /**
-   * @brief Adds information to result message "info" param in case of GENERIC_ERROR
+   * @brief Adds information to result message "info" param in case of
+      GENERIC_ERROR
```

Comment:
@dcherniev Should I close this pull request due to https://github.com/smartdevicelink/sdl_core/pull/1426?